src/Cli/dotnet/Commands/CommandFactory.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+using Microsoft.DotNet.Cli.Commands.Run;
+using Microsoft.DotNet.Cli.Extensions;
+using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.DotNet.Cli.Commands;
+
+public static class CommandFactory
+{
+        internal static CommandBase CreateVirtualOrPhysicalCommand(
```

Comment:
nit: indentation looks wrong
```suggestion
    internal static CommandBase CreateVirtualOrPhysicalCommand(
```